cluster/points_writer.go

```diff
@@ -39,8 +40,27 @@ var (
 
 	// ErrWriteFailed is returned when no writes succeeded.
 	ErrWriteFailed = errors.New("write failed")
+
+	// ErrConsistencyLevelParseError is returned when parsing the string version
+	// of a consistency level.
+	ErrConsistencyLevelParseError = errors.New("write failed")
```

Comment:
Can you rename to `ErrInvalidConsistencyLevel`? Also, the error message doesn't match.